moveit.rosinstall

```diff
@@ -23,7 +27,7 @@
 - git:
     local-name: moveit_tutorials
     uri: https://github.com/ros-planning/moveit_tutorials.git
-    version: master
+    version: pr-master-rework-resources
 - git:
     local-name: panda_moveit_config
```

Comment:
panda_moveit_config is now used from moveit_resources and could be dropped here?!

Reply:
It wasn't used in MoveIt before either, but it's required for the tutorials and thus it's listed in this file.